Packs/FetchIndicatorsFromFile/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py

```diff
@@ -0,0 +1,221 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import re
+import xlrd
+
+
+def xls_file_to_indicator_list(file_path, sheet_name, col_num, starting_row, auto_detect, default_type):
+    indicator_list = []
+    hr_indicators_list = []
+
+    xl_woorkbook = xlrd.open_workbook(file_path)
+    if sheet_name and sheet_name != 'None':
+        xl_sheet = xl_woorkbook.sheet_by_name(sheet_name)
+
+    else:
+        xl_sheet = xl_woorkbook.sheet_by_index(0)
+
+    for row_index in range(0, xl_sheet.nrows):
+        if row_index >= starting_row:
+            indicator = xl_sheet.cell(row_index, col_num).value
+            if auto_detect:
+                indicator_type = detect_type(indicator)
+
+            else:
+                indicator_type = default_type
+
+            hr_indicators_list.append({
+                "Type": indicator_type,
+                "Value": indicator
+            })
+
+            indicator_list.append({
+                'type': indicator_type,
+                'value': indicator
+            })
+
+    return indicator_list, hr_indicators_list
+
+
+def xls_file_to_parsed_indicators(file_path, sheet_name, indicator_col, type_col, starting_row):
+    indicator_list = []
+    hr_indicators_list = []
+
+    xl_woorkbook = xlrd.open_workbook(file_path)
+    if sheet_name and sheet_name != 'None':
+        xl_sheet = xl_woorkbook.sheet_by_name(sheet_name)
+
+    else:
+        xl_sheet = xl_woorkbook.sheet_by_index(0)
+
+    for row_index in range(0, xl_sheet.nrows):
+        if row_index >= starting_row:
+            indicator = xl_sheet.cell(row_index, indicator_col).value
+            indicator_type = xl_sheet.cell(row_index, type_col).value
+
+            hr_indicators_list.append({
+                "Type": indicator_type,
+                "Value": indicator
+            })
```

Comment:
we need just one of those, use only indicator_list it's ok, same in the other func

Reply:
done